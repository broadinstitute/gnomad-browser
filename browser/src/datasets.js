export const referenceGenomeForDataset = datasetId => {
  if (datasetId.startsWith('gnomad_r3')) {
    return 'GRCh38'
  }

  return 'GRCh37'
}

export const datasetLabels = {
  exac: 'ExAC v1.0',
  gnomad_r2_1: 'gnomAD v2.1.1',
  gnomad_r2_1_controls: 'gnomAD v2.1.1 (controls)',
  gnomad_r2_1_non_cancer: 'gnomAD v2.1.1 (non-cancer)',
  gnomad_r2_1_non_neuro: 'gnomAD v2.1.1 (non-neuro)',
  gnomad_r2_1_non_topmed: 'gnomAD v2.1.1 (non-TOPMed)',
  gnomad_r3: 'gnomAD v3.1.2',
  gnomad_r3_controls_and_biobanks: 'gnomAD v3.1.2 (controls/biobanks)',
  gnomad_r3_non_cancer: 'gnomAD v3.1.2 (non-cancer)',
  gnomad_r3_non_neuro: 'gnomAD v3.1.2 (non-neuro)',
  gnomad_r3_non_topmed: 'gnomAD v3.1.2 (non-TOPMed)',
  gnomad_r3_non_v2: 'gnomAD v3.1.2 (non-v2)',
  gnomad_sv_r2_1: 'gnomAD SVs v2.1',
  gnomad_sv_r2_1_controls: 'gnomAD SVs v2.1 (controls)',
  gnomad_sv_r2_1_non_neuro: 'gnomAD SVs v2.1 (non-neuro)',
}

export const allDatasetIds = Object.keys(datasetLabels)

export const labelForDataset = datasetId => datasetLabels[datasetId] || 'Unknown'
