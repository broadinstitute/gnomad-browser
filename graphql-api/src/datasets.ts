export const DATASET_LABELS = {
  gnomad_r4: 'gnomAD v4',
  gnomad_cnv_r4: 'gnomAD v4 cnv',
  gnomad_r3: 'gnomAD v3',
  gnomad_r3_controls_and_biobanks: 'gnomAD v3 (controls and biobanks)',
  gnomad_r3_non_cancer: 'gnomAD v3 (non-cancer)',
  gnomad_r3_non_neuro: 'gnomAD v3 (non-neuro)',
  gnomad_r3_non_topmed: 'gnomAD v3 (non-TOPMed)',
  gnomad_r3_non_v2: 'gnomAD v3 (non-v2)',
  gnomad_r2_1: 'gnomAD v2',
  gnomad_r2_1_controls: 'gnomAD v2 (controls)',
  gnomad_r2_1_non_neuro: 'gnomAD v2 (non-neuro)',
  gnomad_r2_1_non_cancer: 'gnomAD v2 (non-cancer)',
  gnomad_r2_1_non_topmed: 'gnomAD v2 (non-TOPMed)',
  exac: 'ExAC',
}

export const DATASET_REFERENCE_GENOMES: Record<string, string> = {
  gnomad_r4: 'GRCh38',
  gnomad_cnv_r4: 'GRCh38',
  gnomad_r3: 'GRCh38',
  gnomad_r3_controls_and_biobanks: 'GRCh38',
  gnomad_r3_non_cancer: 'GRCh38',
  gnomad_r3_non_neuro: 'GRCh38',
  gnomad_r3_non_topmed: 'GRCh38',
  gnomad_r3_non_v2: 'GRCh38',
  gnomad_r2_1: 'GRCh37',
  gnomad_r2_1_controls: 'GRCh37',
  gnomad_r2_1_non_neuro: 'GRCh37',
  gnomad_r2_1_non_cancer: 'GRCh37',
  gnomad_r2_1_non_topmed: 'GRCh37',
  exac: 'GRCh37',
}
