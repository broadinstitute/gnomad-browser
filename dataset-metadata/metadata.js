import { datasetLabels } from '../browser/src/datasets'

const fullDatasetIds = Object.getOwnPropertyNames(datasetLabels).filter(
  (datasetId) => datasetId === 'exac' || datasetId.match(/_r\d+(_\d+)*$/)
)

export const isSubset = (datasetId) => !fullDatasetIds.includes(datasetId)
