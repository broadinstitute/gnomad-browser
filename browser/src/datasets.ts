import { datasetLabels } from '../../dataset-metadata/metadata'

export const referenceGenomeForDataset = (datasetId: any) => {
  if (datasetId.startsWith('gnomad_r3')) {
    return 'GRCh38'
  }

  return 'GRCh37'
}

// @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
export const labelForDataset = (datasetId: any) => datasetLabels[datasetId] || 'Unknown'
