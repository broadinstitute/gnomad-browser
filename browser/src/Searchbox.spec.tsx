import React from 'react'
import { describe, test, expect } from '@jest/globals'
import { forAllDatasets } from '../../tests/__helpers__/datasets'
import { withDummyRouter } from '../../tests/__helpers__/router'
import renderer from 'react-test-renderer'
import { createBrowserHistory } from 'history'

import Searchbox from './Searchbox'
import { DatasetId } from '@gnomad/dataset-metadata/metadata'

describe('Searchbox', () => {
  test('has no unexpected changes when no dataset specified', () => {
    const tree = renderer.create(withDummyRouter(<Searchbox />))
    expect(tree).toMatchSnapshot()
  })

  forAllDatasets('with selected dataset %s', (datasetId) => {
    test('has no unexpected changes', () => {
      const history = createBrowserHistory()
      history.push(`/?dataset=${datasetId}`)
      const tree = renderer.create(withDummyRouter(<Searchbox history={history} />))
      expect(tree).toMatchSnapshot()
    })

    const expectedDefaultDatasets: Record<DatasetId, DatasetId> = {
      exac: 'exac',
      gnomad_r2_1: 'gnomad_r2_1',
      gnomad_r2_1_controls: 'gnomad_r2_1',
      gnomad_r2_1_non_cancer: 'gnomad_r2_1',
      gnomad_r2_1_non_neuro: 'gnomad_r2_1',
      gnomad_r2_1_non_topmed: 'gnomad_r2_1',
      gnomad_r3: 'gnomad_r3',
      gnomad_r3_controls_and_biobanks: 'gnomad_r3',
      gnomad_r3_non_cancer: 'gnomad_r3',
      gnomad_r3_non_neuro: 'gnomad_r3',
      gnomad_r3_non_topmed: 'gnomad_r3',
      gnomad_r3_non_v2: 'gnomad_r3',
      gnomad_sv_r2_1: 'gnomad_sv_r2_1',
      gnomad_sv_r2_1_controls: 'gnomad_sv_r2_1',
      gnomad_sv_r2_1_non_neuro: 'gnomad_sv_r2_1',
      gnomad_sv_r3: 'gnomad_sv_r3',
    }

    test('has correct default dataset', () => {
      const expectedDefaultDataset = expectedDefaultDatasets[datasetId]
      const history = createBrowserHistory()
      history.push(`/?dataset=${datasetId}`)
      const tree = renderer.create(withDummyRouter(<Searchbox history={history} />))
      const datasetSelect = tree.root.findByType('select')
      const defaultDataset = datasetSelect.props['value']
      expect(defaultDataset).toEqual(expectedDefaultDataset)
    })
  })
})
