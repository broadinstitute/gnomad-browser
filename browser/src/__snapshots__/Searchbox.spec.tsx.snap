// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbox has no unexpected changes when no dataset specified 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset exac has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="exac"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r2_1 has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r2_1_controls has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r2_1_non_cancer has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r2_1_non_neuro has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r2_1_non_topmed has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r3 has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r3"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r3_controls_and_biobanks has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r3"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r3_non_cancer has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r3"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r3_non_neuro has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r3"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r3_non_topmed has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r3"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_r3_non_v2 has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_r3"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_sv_r2_1 has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_sv_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_sv_r2_1_controls has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_sv_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_sv_r2_1_non_neuro has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_sv_r2_1"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;

exports[`Searchbox with selected dataset gnomad_sv_r4 has no unexpected changes 1`] = `
<div
  className="Searchbox__Wrapper-sc-oq7bs2-0 gRnmIj"
>
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    onChange={[Function]}
    value="gnomad_sv_r4"
  >
    <optgroup
      label="GRCh38"
    >
      <option
        value="gnomad_r3"
      >
        gnomAD v3.1.2
      </option>
      <option
        value="gnomad_sv_r4"
      >
        gnomAD SVs v4
      </option>
    </optgroup>
    <optgroup
      label="GRCh37"
    >
      <option
        value="gnomad_r2_1"
      >
        gnomAD v2.1.1
      </option>
      <option
        value="gnomad_sv_r2_1"
      >
        gnomAD SVs v2.1
      </option>
      <option
        value="exac"
      >
        ExAC
      </option>
    </optgroup>
  </select>
  <span
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        {
          "display": "inline-block",
          "width": "100%",
        }
      }
    >
      <input
        aria-autocomplete="list"
        aria-expanded={false}
        autoComplete="off"
        className="Combobox__Input-is1dz3-0 Searchbox__SearchboxInput-kp6w3x-0 iGzWys"
        data-testid="searchbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search by gene, region, or variant"
        role="combobox"
        value=""
      />
    </div>
  </span>
</div>
`;
